beezee/chain.json

```diff
@@ -32,33 +32,33 @@
   },
   "codebase": {
     "git_repo": "https://github.com/bze-alphateam/bze",
-    "recommended_version": "v6.0.0",
+    "recommended_version": "v6.1.0",
     "compatible_versions": [
-      "v6.0.0"
+      "v6.1.0"
     ],
     "binaries": {
-      "darwin/amd64": "https://github.com/bze-alphateam/bze/releases/download/v6.0.0/bze-6.0.0-darwin-amd64.tar.gz",
-      "darwin/arm64": "https://github.com/bze-alphateam/bze/releases/download/v6.0.0/bze-6.0.0-darwin-arm64.tar.gz",
-      "linux/amd64": "https://github.com/bze-alphateam/bze/releases/download/v6.0.0/bze-6.0.0-linux-amd64.tar.gz",
-      "linux/arm64": "https://github.com/bze-alphateam/bze/releases/download/v6.0.0/bze-6.0.0-linux-arm64.tar.gz",
-      "windows/amd64": "https://github.com/bze-alphateam/bze/releases/download/v6.0.0/bze-6.0.0-win64.zip"
+      "darwin/amd64": "https://github.com/bze-alphateam/bze/releases/download/v6.1.0/bze-6.1.0-darwin-amd64.tar.gz",
+      "darwin/arm64": "https://github.com/bze-alphateam/bze/releases/download/v6.1.0/bze-6.1.0-darwin-arm64.tar.gz",
+      "linux/amd64": "https://github.com/bze-alphateam/bze/releases/download/v6.1.0/bze-6.1.0-linux-amd64.tar.gz",
+      "linux/arm64": "https://github.com/bze-alphateam/bze/releases/download/v6.1.0/bze-6.1.0-linux-arm64.tar.gz",
+      "windows/amd64": "https://github.com/bze-alphateam/bze/releases/download/v6.1.0/bze-6.1.0-win64.zip"
     },
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/bze-alphateam/bze/main/genesis.json"
     },
     "versions": [
       {
-        "name": "v6.0.0",
-        "recommended_version": "v6.0.0",
+        "name": "v6.1.0",
```

Comment:
Within the versions array, please add the next_version_name property and instead add a new version to the bottom of the list

Reply:
checked cosmoshub to really understand what you meant. Basically versions should be the exact "description" on how to sync from scratch. Will modify as required asap.

Reply:
Done! Thanks for the help.